### PR TITLE
feat: add Twitter post notification on new blog article

### PR DESCRIPTION
**Pull request by dev-liro**
I'm not exactly too sure about this commit, but could the maintainers test it please ?

### CODE REVIEW

**Comment by dev-liro** [comment]
I forgot to mention, but this PR concerns issue #50

**Review by Sanix-Darker — COMMENTED** [review]
Thanks for this PR, i left some small comments for you !

File: .github/workflows/blog-deploy.yaml [hunk 1]
```diff
@@ -99,3 +99,38 @@ jobs:
             to: ${{ secrets.TELEGRAM_OSSCAMEROON_CHANNEL_ID }}
             token: ${{ secrets.TELEGRAM_BOT_TOKEN }}
             message_file: message
+        - name: Get tweet content
```
[the remark this answers is not in the document]
Comment (dev-liro) [reply .github/workflows/blog-deploy.yaml:102]: This makes more sense

**Comment by elhmn** [comment]
@elhmn note to myself 👉  review this at some point

**Pull request closed without merging**